internal/sqlutil/drivername.go

```diff
@@ -41,6 +41,10 @@ func DriverName(d driver.Driver) string {
 		if strings.HasSuffix(t.PkgPath(), "github.com/lib/pq") {
 			return "postgresql"
 		}
+
+		if strings.HasSuffix(t.PkgPath(), "github.com/jackc/pgx/v4/stdlib") {
```

Comment:
```suggestion
		if strings.HasSuffix(t.PkgPath(), "github.com/jackc/pgx/") {
```

I think that would be safe and future proof, WDYT?